data/brands/shop/mall.json

```diff
@@ -28,19 +28,6 @@
         "brand": "GPT Group",
         "brand:wikidata": "Q3100507",
         "brand:wikipedia": "en:GPT Group",
-        "name": "GPT Group",
```

Comment:
I don't think this is a good idea to remove ```"name"```

According to the discussion by @bhousel in #4985 , at the head of this json have a ```"preserveTags": ["^name"],```, so ```"name"``` are encouraged to changed by user. (You can see [this comment](https://github.com/osmlab/name-suggestion-index/pull/4985#issuecomment-803594165))

so we just offer a name to be rendered and let user know what is it —— a shopping mall, and if they know, they can use a special and specific name to rewrite the ```“name”```

Reply:
Ok, i've added the name tags back.   
The maill.josn file already has a preserveTags at the top so should be ok.